internal/blockchain/v0/reactor.go

```diff
@@ -301,7 +306,7 @@ func (r *Reactor) processBlockchainCh() {
 				}
 			}
 
-		case envelop := <-r.peerUpdatesCh:
+		case envelop := <-r.blockchainOutBridgeCh:
```

Comment:
I know this isn't an artifact of this PR, but can we quickly fix the spelling here please?